lib/octicons_node/script/build.js

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env node
+
+console.log("Create a build/data.json from the exported assets")
```

Comment:
TODO: Right now the figma-action is specific to octicons and is still building the build/data.json for us. I'd like to make it more generic and do the conversion work here. Until then